source/operations/install-deploy-manage/deploy-minio-tenant.rst

```diff
@@ -110,442 +110,528 @@ volume and a supporting
+          By default each service is visible only within the Kubernetes cluster. 
+          Applications deployed inside the cluster can access the services using the ``CLUSTER-IP``. 
+
+          Applications external to the Kubernetes cluster can access the services using the ``EXTERNAL-IP``. 
+          This value is only populated for Kubernetes clusters configured for Ingress or a similar network access service. 
+          Kubernetes provides multiple options for configuring external access to services. 
+          See the Kubernetes documentation on 
+          :kube-docs:`Publishing Services (ServiceTypes) <concepts/services-networking/service/#publishing-services-service-types>` 
+          and :kube-docs:`Ingress <concepts/services-networking/ingress/>` 
+          for more complete information on configuring external access to services.
+
+
+
+   .. tab-item:: Command Line (CLI)
+
+      The :mc:`kubectl minio tenant create` command supports creating a MinIO Tenant in your Kubernetes cluster.
+      The command *requires* that the cluster have a functional MinIO Operator installation.
+
+      You can also use this procedure to generate a Kubernetes ``YAML``-formatted resource document for further customization.
+
+      Prerequisite
+        Initialize the MinIO Operator.
+
+      1) Create a Tenant with default options
+   
+         Use the :mc:`kubectl minio tenant create` command to add a new tenant.
+
+         This command requires the following configurations:
+
+         - :mc:`~kubectl minio tenant create TENANT_NAME`
+     
+           The name to use for the new tenant.
+
+         - :mc:`~kubectl minio tenant create --capacity`
+
+           The total raw storage size for the Tenant. 
+           Specify both the total storage size *and* the :guilabel:`Unit` of that storage. 
+           All storage units are in SI values, e.g. Gi = GiB = 1024\ :sup:`3` bytes.
+     
+           The total storage capacity for the tenant to have across all volumes.
+           The value should be an integer and a standard storage abbreviation, such as ``Gi`` or ``Ti``.
+
+           For example, 16 Ti for 16 Tibibytes.
+
+         - :mc:`~kubectl minio tenant create --servers`
 
-The remaining services support Tenant operations and are not intended for 
-consumption by users or administrators.
+           The total number of MinIO server pods to deploy in the Tenant.
+    
+           The Operator by default uses pod anti-affinity, such that the Kubernetes cluster *must* have at least one worker node per MinIO server pod.
+
+         - :mc:`~kubectl minio tenant create --volumes`
+
+           The total number of storage volumes (Persistent Volume Claims). 
+           The Operator generates an equal number of PVC *plus two* for supporting Tenant services (Metrics and Log Search).
+       
+           The specified :guilabel:`Storage Class` *must* correspond to a set of Persistent Volumes sufficient in number to match each generated PVC.
```

Comment:
This feels a little weird - the user at this point does not know how to specify this. Maybe sub with a reference to the `--storage-class` arg and lead with "Specify a storage class using <arg>`, or omit to use the `default` Kubernetes storage class.